onnxruntime/python/onnxruntime_pybind_ortvalue.cc

```diff
@@ -71,7 +71,52 @@ void addOrtValueMethods(pybind11::module& m) {
 
         return ml_value;
       })
+      .def("update_inplace", [](OrtValue* ml_value, const py::array& py_values) {
```

Comment:
Looks like this is the equivalent of Torch.Tensor's `copy_()` API ? Should we just name it `copy_from_numpy()` to reflect what is being done ?

Reply:
The underscore in `copy_()`  means to update the values in place. So `update_inplace` may be more accurate to describe what the internal is doing?

Reply:
Sure, the name is okay